src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Payment/PaymentReversalTests.cs

```diff
@@ -67,7 +69,12 @@ public async Task Payment_Swish_Reversal(Product[] products, PayexInfo payexInfo
             while (swishPayment.PaymentResponse.Transactions.TransactionList.First(x => x.Type == TransactionType.Reversal).State != State.Completed && counter <= 15)
             {
                 Thread.Sleep(1000);
-                swishPayment = await SwedbankPayClient.Payments.SwishPayments.Get(paymentLink, PaymentExpand.All);
+                try
+                {
+                    swishPayment = await SwedbankPayClient.Payments.SwishPayments.Get(paymentLink, PaymentExpand.All);
+                }
+                catch (HttpResponseException){}

```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6000212)